web/src/cypress/integration/favorites.ts

```diff
@@ -113,5 +105,35 @@ function check(
           .should('contain', name1)
       })
     }
+
+    if (typeName === 'Service') {
```

Comment:
I'll take a shot at making this more generic -- otherwise, it probably doesn't belong here, since we're already verifying the list page and this is very specific to the alert flow.